infra/presubmit.py

```diff
@@ -226,10 +227,45 @@ def check_project_yaml(paths):
   return all([_check_one_project_yaml(path) for path in paths])
 
 
+def _check_one_options_file(options_file_path):
+  """Checks that a single options file is valid."""
+  if os.path.basename(os.path.dirname(
+      os.path.dirname(options_file_path))) != 'projects':
+    # Is this a project file?
+    return True
+
+  _, extension = os.path.splitext(options_file_path)
+  if extension != '.options':
+    # Don't check it if it isn't an options file.
+    return True
+
+  config = configparser.ConfigParser()
+  config.read(options_file_path)
+
+  try:
+    config.get('libfuzzer', 'max_len')
+    print(('Options file: {file_path} is invalid. max_len cannot be enforced '
+           'by honggfuzz and AFL. Enforce this in code instead.').format(
```

Comment:
in fuzzer code with a check on |size| field (to be more descriptive or similar)?

Reply:
Perhaps we can add a link to an example?

Reply:
I now link to this issue where I describe the problem and solution:
https://github.com/google/oss-fuzz/issues/5421